tests/app/auth/login/login.component.spec.ts

```diff
@@ -36,10 +36,45 @@ describe('LoginComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should start login process', () => {
-    component.startLoginProcess();
+  it('should call startLoginProcess with false in startPublicOnlyLoginProcess', () => {
```

Comment:
Spec description is a bit confusing, maybe can rename it to something like `startPublicOnlyLoginProcess should call startLoginProcess with false`. Similar comment for the test spec which calls `startIncludePrivateLoginProcess` in line 47